python.d/smartd_log.chart.py

```diff
@@ -197,11 +197,12 @@ def create_lines(attrid):
                 result.append(['_'.join([name, attrid]), name[:name.index('.')], 'absolute'])
             return result
 
-        # Add additional smart attributes to the ORDER. If something goes wrong we don't care.
-        try:
-            ORDER.extend(list(set(self.attr.split()) & SMART_ATTR.keys() - set(ORDER)))
-        except Exception:
-            pass
+        # Use configured attributes, if present. If something goes wrong we don't care.
+        if self.attr:
+            try:
+                ORDER = [attr for attr in self.attr.split() if attr in SMART_ATTR.keys()]
+            except Exception:
+                pass
         self.order = [''.join(['attrid', i]) for i in ORDER]
         self.definitions = dict()
```

Comment:
I am ok with replacing extend with custom order. But lets do additional check. I mean if (custom)ORDER is empty list.
```python
ORDER = [attr for attr in self.attr.split() if attr in SMART_ATTR.keys()] or ORDER
```